feature/search/src/main/java/kr/co/cotton/feature/search/SearchViewModel.kt

```diff
@@ -31,10 +31,14 @@ class SearchViewModel @Inject constructor(
                 }
             }.collect()
     }
+
+    fun onClickCard(searchResult: SearchResult) {
+
```

Comment:
🚫 **[ktlint]** <no-empty-first-line-in-method-block> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>First line in a method block should not be empty